src/start/PasswordAuthScreen.js

```diff
@@ -89,7 +89,7 @@ class PasswordAuthView extends PureComponent<Props, State> {
     const isButtonDisabled = password.length === 0 || !isValidEmailFormat(email);
 
     return (
-      <Screen title="Log in" padding centerContent>
+      <Screen title="Log in" centerContent keyboardShouldPersistTaps="always" padding>
```

Comment:
Likewise, `centerContent` and `padding` are good to keep next to each other, because their functions are closely related.